server_test.go

```diff
@@ -570,6 +570,37 @@ func TestServer_NonExistingUsers(t *testing.T) {
 	}
 }
 
+// Ensure the server does return valid users grant data
+func TestServer_GetGrantPrivileges(t *testing.T) {
```

Comment:
We're migrating tests to `cmd/influxd/server_integration_test.go`.  Can you remove this one and add tests to the table in that file?  The table is in the `runTestsData` func.